guide/additional-info/changes-in-v12.md

```diff
@@ -375,17 +430,24 @@ Just like the `Channel#send***` methods, all the `.send***()` methods have been
 
 #### ClientUser#setPassword
 
-The second parameter in `clientUser.setPassword()` has been changed. The `oldPassword` parameter from v11 has been changed into an object as the second parameter.
-
-```diff
-- clientUser.setPassword('newpassword', 'oldpassword');
-+ clientUser.setPassword('newpassword', { oldPassword: 'oldpassword', mfaCode: '123456' });
-```
+`clientUser.setPassword()` has been removed entirely.
 
 #### ClientUser#unblock
 
 `clientUser.unblock()` has been removed entirely.
 
+### ClientUserChannelOverride
+
+The `clientUserChannelOverride` class has been removed entirely.
+
+### ClientUserGuildSettings
+
+The `clientUserGuildSettings` class has been removed entirely.
+
+### ClientUserSettings
+
+The `clientUserSettings` class has been removed entirely.
```

Comment:
Class name should start with an uppercase letter `ClientUserSettings`

Reply:
Fixed the various class capitalization mistakes.